src/viam/sessions_client.py

```diff
@@ -47,13 +49,15 @@ class SessionsClient:
 
     _HEARTBEAT_MONITORED_METHODS: MutableMapping[str, bool] = {}
 
-    def __init__(self, channel: Channel, direct_dial_address: str, dial_options: Optional[DialOptions], *, disabled: bool = False):
+    def __init__(self, channel: Channel, direct_dial_address: str, dial_options: Optional[DialOptions], *, disabled: bool = False, robot_addr: Optional[str] = None):
         self.channel = channel
         self.client = RobotServiceStub(channel)
         self._address = direct_dial_address
+        self._robot_address = robot_addr
         self._disabled = disabled
         self._dial_options = deepcopy(dial_options) if dial_options is not None else DialOptions()
-        self._dial_options.disable_webrtc = True
+        if sys.platform != "win32" and sys.platform != "cygwin":
+            self._dial_options.disable_webrtc = True
```

Comment:
I was unable to get the `SessionsClient` to reliably connect _and stay connected_ on windows when webRTC was disabled. Leaving it enabled is a little inefficient insofar as it means we're creating a second rust runtime, but 1) the windows workflow is already a little inefficient because we're using TCP instead of UDS for our proxy, and 2) I figure a minor inefficiency here is better than not supporting a `SessionsClient` on windows at all. No regression on linux/mac since we leave it disabled when the platform isn't windows.

cc @cheukt, who set up the disabling of webRTC initially [here](https://github.com/viamrobotics/viam-python-sdk/pull/447). If there's a particular reason to be concerned about re-enabling, please let me know!